docs(api): add `<NuxtWelcome />` component docs (#6745)
nuxt/framework: docs/content/3.api/2.components/6-nuxt-welcome.md

# `<NuxtWelcome>`

::alert{type=info icon=🔎}

> This component is part of [@nuxt/ui](https://github.com/nuxt/ui)

Reviewer: ```suggestion
This component is part of [@nuxt/ui](https://github.com/nuxt/ui).
```

# `<NuxtWelcome>`

::alert{type=info icon=🔎}
This component is part of [@nuxt/ui](https://github.com/nuxt/ui)
::

> Nuxt uses the `<NuxtWelcome>` component to greet users in new projects. It includes links to the Nuxt documentation, source code and social media accounts.

Reviewer 2: ```suggestion
Nuxt uses the `<NuxtWelcome>` component to greet users in new projects made from the starter template. It includes links to the Nuxt documentation, source code, and social media accounts.
```

# `<NuxtWelcome>`

> Nuxt uses the `<NuxtWelcome>` component to greet users when framework initialized. It provides information about latest version of framework, documentation link and how to start building quickly.

Reviewer 3: ```suggestion
Nuxt uses the `<NuxtWelcome>` component to greet users in new projects. It includes links to the Nuxt documentation, source code and social media accounts.
```
